src/main/java/salesforce/ui/pages/accounts/NewAccountPopup.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+
+package salesforce.ui.pages.accounts;
+
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import salesforce.ui.pages.BasePage;
+
+import java.util.HashMap;
+
+public class NewAccountPopup extends BasePage {
+
```

Comment:
I think you are not using this web element anymore. You should delete it. The same goes for faxTxtBox